packages/ia-components/sandbox/track-lists/track.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Creates the track title to display based on track object
+ *
+ * @param { string } name
+ * @param { string } title
+ * @param { string } artist
+ * @param { string } creator
+ * @param { boolean } isAlbum
+ *
+ * @return { string or react fragment } trackTitle
+ */
+const parseTrackTitle = ({
```

Comment:
I don't know if we have a standard for this already, but I personally prefer having any stateless functional React components named in TitleCase to match React component names created by subclassing `React.Component`. Then if you use it in JSX, it matches naming convention with stateful components.

Reply:
thanks, got this updated.